plugins/common/glide.lock

```diff
@@ -0,0 +1,8 @@
+hash: e438ca8cf1bdca3e9c984bb1ee70c0d58be9da52cbe13a079a7c6cbc14210a37
```

Comment:
Glide is not a requirement. It's just the tool I chose. Any golang 1.5+ dep tool is fine